docs/reference/hacl/hpke/index.md

```diff
@@ -1,3 +1,407 @@
 # HPKE
 
-...
+Hybrid public key encryption (HPKE) according to [RFC 9180].
```

Comment:
Please add something to the effect that this is experimental/unstable as discussed.